web/frontend/src/pages/admin/DKGTable.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import ProxyRow from './ProxyRow';
+import * as endpoints from 'components/utils/Endpoints';
+import useFetchCall from 'components/utils/useFetchCall';
+import { FlashContext, FlashLevel } from 'index';
+import Loading from 'pages/Loading';
+
+const DKGTable: FC = () => {
```

Comment:
Should be added on the admin page.
It is missing a button to add and delete a mapping.

Reply:
Right I'm doing it right now 😁